coverage/docs/design.md

```diff
@@ -0,0 +1,113 @@
+#Overview
```

Comment:
This doesn't mention the use of GCS for persistent storage. I think that is important to mention since the job fetches results from previous runs from GCS.

Reply:
Done.